src/services/apps-metadata.ts

```diff
@@ -16,6 +20,7 @@ export type MetadataCommon = {
     url: Url;
     appColor: string;
     hiddenInAppsMenu: boolean;
+    defaultCountry?: string;
```

Comment:
It's only in Study metadata no ? Why in commons ?

Reply:
Wasn't aware it's not a common one, I moved it to MetadataStudy